client/src/components/ExtraUsers/ExtraUsers.js

```diff
@@ -0,0 +1,54 @@
+
+import React, { Component } from 'react';
+import { Link } from "react-router-dom";
+import "./ExtraUsers.css";
+
+
+// Class Based React Component
+class ExtraUsers extends Component {
```

Comment:
I don't know that this component needs state - I think it could be a function component